### PR TITLE
Pylint

### PR DESCRIPTION
**Pull request by RunDevelopment**

### CODE REVIEW

**Comment by RunDevelopment** [comment]
Failed as expected. Good.

**Comment by RunDevelopment** [comment]
There are still a bunch of warning left btw. We'll have to resolve those over time. With "we" I mean someone that is not me. I just don't know enough about Python to fix those warnings.

File: requirements.txt [hunk 1]
```diff
@@ -1,7 +1,10 @@
...
 numpy
+opencv-python
 Pillow
-ncnn_vulkan
-sanic_cors
+pyinstaller
+pylint
 sanic
+sanic_cors
+websockets==9.1
```
Comment (joeyballentine) [inline requirements.txt:9]: Websockets isn't necessary

Comment (joeyballentine) [reply requirements.txt:9]: Sorry I think I misled you with the old one

Comment (joeyballentine) [reply requirements.txt:9]: Tbh, not a problem.

**Pull request merged**